torch/optim/rmsprop.py

```diff
@@ -369,6 +421,14 @@ def _multi_tensor_rmsprop(
         if momentum > 0:
             torch._foreach_mul_(grouped_momentum_buffer_list, momentum)
             torch._foreach_addcdiv_(grouped_momentum_buffer_list, grouped_grads, avg)
-            torch._foreach_add_(grouped_params, grouped_momentum_buffer_list, alpha=-lr)
+            if capturable:
```

Comment:
```suggestion
            if capturable and isinstance(lr, Tensor):
```
and everywhere else too